extendr-api/tests/optional/either/either_robj_from_tests.rs

```diff
@@ -0,0 +1,70 @@
+use either::Either::{self, Left, Right};
+use extendr_api::prelude::*;
+use rstest::rstest;
+
+#[rstest]
+#[case("1L", Rint::from(1i32), "")]
+#[case("1.5", Rfloat::from(1.5), Rint::default())]
+#[case("\"string\"", "string", false)]
+#[case("TRUE", true, 0)]
+#[case("1 + 2i", Rcplx::new(1.0, 2.0), 0)]
```

Comment:
What happens if you use the true "omit me" type in Rust i.e. `()` for `_TRight` here?

Reply:
No longer applicable